src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private int index;
+
+    public Color getRandomColor() {
+        index = new Random().nextInt(Color.values().length);
```

Comment:
Every invoke of method you create new object of Random, save it in field of class